initrd/dockerfile.go

```diff
@@ -126,6 +126,10 @@ func NewFromDockerfile(ctx context.Context, path string, opts ...InitrdOption) (
 		return nil, fmt.Errorf("file is not a Dockerfile")
 	}
 
+	if _, err := os.Stat(path); err != nil {
```

Comment:
This will actually break building outside the ``cwd``.
E.g.: ``kraft build ./example``

This raises an issue about what "path" means in the context of this initrd implementation. The root problem is actually how the ramfs is constructed [here](https://github.com/unikraft/kraftkit/blob/staging/internal/cli/kraft/run/utils.go#L367). Path is not relative to the cwd, but instead it is relative to the working directory, which is only seeded after the opts are being applied.

A fix would be to move the check after the opts and concatenate the path to the workdir, or maybe the whole thing should be redesigned a bit such that path is relative to the cwd?

Reply:
Hmm, no idea, in the error that triggered me to fix this, the path had the given workdir also

Also, this check is already present in the file/directory anyway so i thought to match

Still the second commit should go in as is because the order is currently not correct

Reply:
Remove the stat and keep the re-order; update commit message about re-ordering.